widgets/use-calcite-components/README.md

```diff
@@ -0,0 +1,43 @@
+# Use calcite components widget
+
+This sample demonstrates how to use [Calcite components](https://developers.arcgis.com/calcite-design-system/components/) in a custom widget.
+
+## Reminder before use
+
+Before you use calcite components, we suggest you review the components in the jimu-ui package. Jimu-ui is the primary component library for Experience Builder, and you may want to use that first because it will keep your custom widgets and themes consistent with the rest of the Experience Builder theme. If you do have a requirement to use calcite components, this example will help you.
```

Comment:
```suggestion
Before you use Calcite components, we suggest you review the components in the jimu-ui package. Jimu-ui is the primary component library for the developer edition of ArcGIS Experience Builder, and you may want to use that first because it will keep your custom widgets and themes consistent with the rest of the Experience Builder theme. If you do have a requirement to use Calcite components, this example will help you.
```